src/TypeSpec/__Private/KeyedTraversableSpec.hack

```diff
@@ -72,4 +73,13 @@ extends TypeSpec<T> {
     }
     return /* HH_IGNORE_ERROR[4110] */ $value;
   }
+
+  <<__Override>>
+  public function toString(): string {
+    return Str\format(
+      'KeyedTraversable<%s, %s>',
```

Comment:
nit: use `::class` vs string literals consistently (everything else seems to be using `::class` but make sure whatever we use gets namespaces expanded/ignored for autoimported types the way we want)